packages/camera/camera/ios/Classes/CameraPlugin.m

```diff
@@ -992,6 +992,15 @@ - (void)handleMethodCallAsync:(FlutterMethodCall *)call result:(FlutterResult)re
     NSDictionary *argsMap = call.arguments;
     NSUInteger cameraId = ((NSNumber *)argsMap[@"cameraId"]).unsignedIntegerValue;
     if ([@"initialize" isEqualToString:call.method]) {
+    NSString *videoFormatValue = ((NSString *)argsMap[@"imageFormatGroup"]);
+    if ([videoFormatValue isEqualToString:@"bgra8888"]) {
+        videoFormat = kCVPixelFormatType_32BGRA;
+    } else if ([videoFormatValue isEqualToString:@"yuv420"]) {
+        videoFormat = kCVPixelFormatType_420YpCbCr8BiPlanarVideoRange;
+    } else {
+        NSLog(@"The selected imageFormatGroup is not supported by iOS. Defaulting to brga8888");
+        videoFormat = kCVPixelFormatType_32BGRA;
+    }
```

Comment:
This should be moved into its own method (similar to the `getFlashModeForString`), makes the code easier to read.